package.json

```diff
@@ -10,10 +10,8 @@
   "scripts": {
     "commit": "git-cz",
     "prebuild": "rimraf dist",
-    "start:dev": "webpack-dev-server --mode development --watch-content-base --open",
-    "build": "npm-run-all build:*",
-    "build:main": "webpack --output-filename ml5.js",
-    "build:min": "uglifyjs dist/ml5.js -o dist/ml5.min.js",
+    "start": "webpack-dev-server --open --config webpack.dev.babel.js",
```

Comment:
The `--open` flag launches a web browser in `localhost:8080` but with nothing on it. It might be better to remove this because it might confuse begginers. @oveddan what do you think is best practice in this case?

Reply:
Yes this stumped me....I think we should remove it unless there is some demo home page.

Reply:
Maybe that is a good idea. Whenever someone does runs `npm start` the browser opens to a page with a few instructions on how to start contributing. If we wanted something like that, where should that file be placed?

Reply:
The instructions are a great idea. I'm not sure yet how to add that but will do the necessary research to figure that out :)